recipes-wolfssl/wolfssl/commercial/README.md

```diff
@@ -0,0 +1,105 @@
+# Yocto wolfSSL FIPS and Commerical Setup Instructions
+
+## Prerequisites
+
+- Yocto environment is set up and ready.
+
+## Steps
+
+1. **Clone the meta-wolfSSL Repository**
```

Comment:
`meta-wolfSSL` should be `meta-wolfssl` for consistent case sensitivity as repo name.

Please apply this to other places this happens in this file as well, thanks.